core/google/cloud/_testing.py

```diff
@@ -68,8 +66,12 @@ def _tempdir_mgr():
     return _tempdir_mgr
 
 
+# pylint: disable=invalid-name
+# Retain _tempdir as a constant for backwards compatibility despite
```

Comment:
I will as part of the core v1.0.0 refactor I proposed, but I wanted to do this without breaking any interfaces.